src/core/components/odsCoreProvider/odsCoreProvider.mdx

```diff
@@ -5,12 +5,13 @@ import { OdsCoreProvider } from './odsCoreProvider';
 
 <Title>OdsCoreProvider</Title>
 
-The `<OdsCoreProvider />` is a React context that defines the image and link elements to be rendered across all the ODS
-components, defaulting such values to `<img />` and `<a />` tags. The provider is used by the `<AvatarBase />` and
-`<LinkBase />` components, which are in turn used by all the other core and module components.
+The `<OdsCoreProvider />` is a React context that defines the image, link elements, and copy texts to be rendered across
+all the ODS components, defaulting such values to `<img />`, `<a />` tags, and a default set of copy texts. The provider
+is used by the `<AvatarBase />` and `<LinkBase />` components, which are in turn used by all the other core and module
```

Comment:
> The provider is used by the `<AvatarBase />` and `<LinkBase />` component

This is wrong now, the provider is used by many more components

Reply:
Removed this sentence all together https://github.com/aragon/ods/pull/221/commits/4889d2d8f80058bbc1615400862db79407c6e8f4